app_dart/lib/src/request_handlers/file_flaky_issue_and_pr.dart

```diff
@@ -49,6 +49,7 @@ class FileFlakyIssueAndPR extends ApiRequestHandler<Body> {
     final String testOwnerContent = await gitHub.getFileContent(slug, kTestOwnerPath);
     final Map<String?, Issue> nameToExistingIssue = await getExistingIssues(gitHub, slug);
     final Map<String?, PullRequest> nameToExistingPR = await getExistingPRs(gitHub, slug);
+    int filedIssueAndPRNumber = 0;
```

Comment:
nit: consider renaming this to filedIssueAndPRCount. Number makes me think this is tracking the PR number.

Reply:
Updated.